### PR TITLE
Add new notebooks to the documentation

### PR DESCRIPTION
**Pull request by FlorentinD**

### CODE REVIEW

File: doc/modules/ROOT/pages/tutorials/centrality-algorithms.adoc [hunk 1]
```diff
@@ -0,0 +1,317 @@
+// DO NOT EDIT - AsciiDoc file generated automatically
+
+= Centrality Algorithms
```
Comment (adamnsch) [inline doc/modules/ROOT/pages/tutorials/centrality-algorithms.adoc:3]: Missing colab link? Other page too

**Review by adamnsch — CHANGES_REQUESTED** [review]
Looks good! But are we missing colab links?

File: doc/modules/ROOT/pages/tutorials/centrality-algorithms.adoc (hunk 1 above)
Comment (FlorentinD) [reply doc/modules/ROOT/pages/tutorials/centrality-algorithms.adoc:3]: true, they exist in the notebooks but got lost during conversion :thinking:

Comment (FlorentinD) [reply doc/modules/ROOT/pages/tutorials/centrality-algorithms.adoc:3]: @nvitucci do you have an idea why the cell is ignored? I cant spot the difference to other notebooks

Comment (nvitucci) [reply doc/modules/ROOT/pages/tutorials/centrality-algorithms.adoc:3]: The way the converter works at the moment requires the `colab_type` key to be present in the Colab cell metadata (which is empty in this notebook), like this:

```
"metadata": {
    "colab_type": "text",
    "id": "view-in-github"
   }
```

Comment (FlorentinD) [reply doc/modules/ROOT/pages/tutorials/centrality-algorithms.adoc:3]: thanks that solved it. Should have looked at the raw files and not the VS Code view

**Review by adamnsch — APPROVED** [review]

**Pull request merged**